components/chart/rendering-modes.md

```diff
@@ -24,6 +24,7 @@ Using vector graphics ensures that:
 
 * The browser zoom does not degrade the image.
 * The prints are crisp regardless of the resolution.
+* The SVG Chart provides accessibility features such as [general description]({%slug components/chart/overview%}#chart-title) and [aria label templates]({%slug components/chart/label-template-format%}#series-label-aria-template), which the Canvas Chart does not.
 
 Since `SVG` is the default rendering mode of the Chart, you don't need to explicitly define it.
```

Comment:
I would remove this line, or possibly merge it somehow with the intro line of the section, to remove the redundancy.